debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/RomClassWalker.java

```diff
@@ -687,6 +691,16 @@ void allSlotsInOptionalInfoDo() throws CorruptDataException
 			allSlotsInAnnotationDo(U32Pointer.cast(cursor.get()), "classAnnotations");
 			cursor = cursor.add(1);
 		}
+		if (romClass.optionalFlags().allBitsIn(J9NonbuilderConstants.J9_ROMCLASS_OPTINFO_RECORD_ATTRIBUTE)) {
+			classWalkerCallback.addSlot(clazz, SlotType.J9_SRP, cursor, "RecordAttributeSRP");
```

Comment:
Most calls to `addSlot()` supply a slot name beginning with a lower-case letter: why the departure here and on line 700 below?

Reply:
I will change it to lower case.